_maps/map_files/cyberiad/z3.dmm

```diff
@@ -1,357 +1,234 @@
+"bF" = (/obj/machinery/door/airlock/external{frequency = 1382; icon_state = "door_locked"; id_tag = "telecomms_server_room_outer"; locked = 1; name = "Telecomms Server Room"; req_access = null; req_access_txt = "13"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "dark"; dir = 5},/area/tcommsat/server)
+"bG" = (/obj/machinery/camera,/obj/structure/lattice,/turf/space,/area/turret_protected/tcomsat)
+"bH" = (/obj/machinery/power/apc{dir = 1; name = "Telecoms Sat. APC"; pixel_x = 1; pixel_y = 26},/obj/structure/cable{icon_state = "0-4"; d2 = 4},/turf/simulated/floor/plating/airless/catwalk,/area/turret_protected/tcomsat)
+"bI" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/obj/effect/decal/warning_stripes{icon_state = "N-corner"},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "dark"; dir = 5},/area/turret_protected/tcomsat)
+"bJ" = (/obj/effect/spawner/window/reinforced,/obj/structure/cable{icon_state = "0-4"; d2 = 4},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/turf/simulated/floor/plating,/area/turret_protected/tcomsat)
+"bK" = (/obj/effect/spawner/window/reinforced,/obj/structure/cable{d2 = 8; icon_state = "0-8"},/turf/simulated/floor/plating,/area/turret_protected/tcomsat)
+"bL" = (/obj/structure/lattice,/obj/machinery/camera{c_tag = "Telecomms Server Room North"; dir = 2; network = list("Telecomms"); pixel_x = 20},/turf/space,/area/turret_protected/tcomsat)
+"bM" = (/obj/structure/lattice,/obj/machinery/light{dir = 8},/turf/space,/area/turret_protected/tcomsat)
+"bN" = (/obj/machinery/porta_turret,/turf/simulated/floor/plating,/area/turret_protected/tcomsat)
+"bO" = (/obj/machinery/embedded_controller/radio/airlock/airlock_controller{frequency = 1382; id_tag = "telecomms_server_room"; pixel_x = 25; pixel_y = -5; req_access_txt = "13"; tag_airpump = "telecomms_server_room_vent"; tag_chamber_sensor = "telecomms_server_room_sensor"; tag_exterior_door = "telecomms_server_room_outer"; tag_interior_door = "telecomms_server_room_inner"},/obj/machinery/airlock_sensor{frequency = 1382; id_tag = "telecomms_server_room_sensor"; pixel_x = 25; pixel_y = 5},/obj/machinery/atmospherics/unary/vent_pump/high_volume{dir = 2; frequency = 1382; id_tag = "telecomms_server_room_vent"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; tag = ""},/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "dark"; dir = 5},/area/turret_protected/tcomsat)
```

Comment:
1 minute is far too long if you ask me.

Reply:
No idea how to tweak that, apart from adding more vents

Reply:
Well, yeah, probably the room should be 2*3 with three vents instead?